### PR TITLE
Week 2 - Project News Site

### PR DESCRIPTION
**Pull request by faypi**
I created multiple Flexboxes with different behaviours. 
The header flexbox changes direction based on screen size.
The hero section has a with header and a2 column flexbox with gap.
The featured article flexbox with 2 items that change order when resized to different sizes.
As requested, the "Popular Destinations"section with a responsive flexbox with:

- 4 columns on laptop
- 3 on tablet
- 1 on mobile

For the CSS Styles I followed the mobile-first principle and configured separate styles for tablets and laptops/desktops.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,163 @@
...
+    <!-- Header that changes direction based on screen size -->
+    <div class="nav-container">
+      <div class="logo">
+        <img src="img/Travel..png" width="100px" target="_blank">
+      </div>
+      <div class="navbar">
+        <div class="navbar-item"><a href="#">Home</a></div>
+        <div class="navbar-item"><a href="#">Guides</a></div>
+        <div class="navbar-item"><a href="#">Top Picks</a></div>
+        <div class="navbar-item"><a href="#">Destinations</a></div>
```
Comment (idanaslund) [inline code/index.html:28]: Name of class follows guidlines for good code!

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,163 @@
...
+          </h2>
+          <p>
+            Planning a trip overseas, whether long or short, can seem overwhelming. But it doesn’t have to be! In this
+            article, we’ll show you how to take the stress out of planning a trip, by laying out a simple step-by-step
+            guide that’ll ensure you don’t forget anything. </p>
+          <button class="featured-article-button">Read More</button>
+        </article>
+      </div>
+      <div class="flex-container-2">
+        <img src="img/rolands-varsbergs-miKmVyq3qhE-unsplash.jpg" alt="article-1">
```
Comment (idanaslund) [inline code/index.html:72]: Maybe name of `alt` should explain what is in the picture for someone who cannot see it?

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,163 @@
...
+        <div class="navbar-item"><a href="#">Home</a></div>
+        <div class="navbar-item"><a href="#">Guides</a></div>
+        <div class="navbar-item"><a href="#">Top Picks</a></div>
+        <div class="navbar-item"><a href="#">Destinations</a></div>
+      </div>
+    </div>
+  </header>
+
+  <div class="main-container">
+    <!-- Hero section with header and 2 column flexbox with gap -->
```
Comment (idanaslund) [inline code/index.html:34]: Nice way of explaining this part of the code.

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,163 @@
...
+          </div>
+        </article>
+      </div>
     </section>
-  </body>
-</html>
+
+    <!-- Footer section with responsive flexbox -->
+    <div class="footer">
+      <div class="footer-left">
```
Comment (idanaslund) [inline code/index.html:142]: Maybe confusing class names if you start with mobile first and the three `div`s are placed in a column?

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,262 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* 
+CSS Styles for multiple flexboxes.
+Followed the mobile-first principles and
+configured seperate styles for tablets and laptops/desktops.
```
Comment (idanaslund) [inline code/style.css:4]: Nice description!

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,262 @@
...
+  font-size: 1.6em;
+}
+
+
+.nav-container {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: column;
+  margin-top: 10px;
```
Comment (idanaslund) [inline code/style.css:31]: `margin-top` doesn´t seem to work because of the `margin: 0 auto;` on row 33.

**Review by idanaslund — APPROVED** [review]
Overall a very pretty and functional project. Also good descriptions of the code, it made me keep track of where I was on the page. I left some comments but can´t think of anything else to change or comment at the moment. I learned from reading your code.

File: code/index.html (hunk 4 above)
Comment (faypi) [reply code/index.html:142]: Just saw those Ida!! Thanks, your comment makes sense

File: code/index.html (hunk 2 above)
Comment (faypi) [reply code/index.html:72]: Very good comment. I guess I was a bit lazy with accessibility 🙄

**Pull request closed without merging**